components/header/menu/styles.module.css

```diff
@@ -10,12 +10,12 @@
 
 .dialog {
   position: absolute;
-  padding: 0.75rem;
   min-width: max-content;
   position: absolute;
   z-index: 10;
   visibility: hidden;
   will-change: transform, opacity;
+  padding: 0.75rem !important;
```

Comment:
With Chrome, I have no priority problem in local with this :
<img width="1470" alt="image" src="https://github.com/user-attachments/assets/eb30120c-318c-489c-882c-22742bd30e9b">

Reply:
The weird thing is I dont always have the conflict.